Search/Exception/MetadataNotFoundException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Search\Exception;
+
+class MetadataNotFoundException extends \RuntimeException
```

Comment:
I said it's not useful here. But the information which metadata was not found might be useful in the place the exception is caught. And if it is, I don't want to use some string operations on the exception message to find out.